tests/Facebook/InstantArticles/Transformer/WPTransformerTest.php

```diff
@@ -74,7 +74,7 @@ public function testTransformerLikeWPContent()
         $this->assertEquals($expected, $result);
         // there must be 3 warnings related to <img> inside <li> that is not supported by IA
         // And 1 warning related to the getter
-        $this->assertEquals(4, count($transformer->getWarnings()));
+        $this->assertEquals(3, count($transformer->getWarnings()));
```

Comment:
Nice catch @diegoquinteiro.

Fixed the comments. Since the tests started using the fragment, and the rules were better structured, no getter exception was raised.